code/modules/modular_computers/file_system/programs/messenger/messenger_program.dm

```diff
@@ -759,6 +758,12 @@
 			var/obj/item/modular_computer/pda/comp = computer
 			comp.explode(usr, from_message_menu = TRUE)
 
+/datum/computer_file/program/messenger/proc/compare_name(datum/computer_file/program/messenger/rhs)
+	return sorttext(rhs?.computer?.saved_identification, computer?.saved_identification)
+
+/datum/computer_file/program/messenger/proc/compare_job(datum/computer_file/program/messenger/rhs)
+	return sorttext(rhs?.computer?.saved_job, computer?.saved_job)
```

Comment:
why do we need a ? on rhs?

Reply:
we don't prolly, that's just how `cmp_pdaname_asc` and `cmp_pdajob_asc` did it